services/mirror/mirror.go

```diff
@@ -229,18 +229,14 @@ func runSync(m *models.Mirror) ([]*mirrorSyncResult, bool) {
 		stderrMessage, sanitizeErr := sanitizeOutput(stderr, repoPath)
 		if sanitizeErr != nil {
 			log.Error("sanitizeOutput failed on stderr: %v", sanitizeErr)
-			log.Error("Failed to update mirror repository %v:\nStdout: %s\nStderr: %s\nErr: %v", m.Repo, stdout, stderr, err)
-			return nil, false
 		}
-		stdoutMessage, err := sanitizeOutput(stdout, repoPath)
-		if err != nil {
+		stdoutMessage, sanitizeErr := sanitizeOutput(stdout, repoPath)
+		if sanitizeErr != nil {
 			log.Error("sanitizeOutput failed: %v", sanitizeErr)
-			log.Error("Failed to update mirror repository %v:\nStdout: %s\nStderr: %s\nErr: %v", m.Repo, stdout, stderrMessage, err)
-			return nil, false
 		}
 
 		log.Error("Failed to update mirror repository %v:\nStdout: %s\nStderr: %s\nErr: %v", m.Repo, stdoutMessage, stderrMessage, err)
```

Comment:
`err` may contain credetials too. Is the system log supposed to be santized?

Reply:
Not that i would know, sinse if you can read the log you can also read tge confug & so have access to DB